torchvision/prototype/transforms/_transform.py

```diff
@@ -1,18 +1,19 @@
 import enum
-from typing import Any, Dict, Tuple, Type
+from typing import Any, Callable, Dict, Tuple, Type, Union
 
 import PIL.Image
 import torch
 from torch import nn
 from torch.utils._pytree import tree_flatten, tree_unflatten
 from torchvision.prototype.features import _Feature
+from torchvision.prototype.transforms._utils import _isinstance, is_simple_tensor
 from torchvision.utils import _log_api_usage_once
 
 
 class Transform(nn.Module):
 
     # Class attribute defining transformed types. Other types are passed-through without any transformation
-    _transformed_types: Tuple[Type, ...] = (torch.Tensor, _Feature, PIL.Image.Image)
+    _transformed_types: Tuple[Union[Type, Callable[[Any], bool]], ...] = (is_simple_tensor, _Feature, PIL.Image.Image)
```

Comment:
I think that's correct. @pmeier thoughts?

Reply:
We need to transform data as pure torch.Tensor or Image or BBox, Mask, Labels (all _Features) and PIL images. We could either keep `torch.Tensor` and all `_Feature` then would pass automatically and also other user tensor subclasses which we are not aware of.

Reply:
We discussed offline, that's correct indeed.